src/cata_tiles.cpp

```diff
@@ -1855,11 +1859,45 @@ bool cata_tiles::draw_from_id_string( const std::string &id, TILE_CATEGORY categ
             case LINE_XXXX:
                 sym = LINE_XXXX_C;
                 break;
+            case LINE_XOXO_UNICODE:
+                sym = LINE_XOXO_C;
+                break;
+            case LINE_OXOX_UNICODE:
+                sym = LINE_OXOX_C;
+                break;
+            case LINE_XXOO_UNICODE:
+                sym = LINE_XXOO_C;
+                break;
+            case LINE_OXXO_UNICODE:
+                sym = LINE_OXXO_C;
+                break;
+            case LINE_OOXX_UNICODE:
+                sym = LINE_OOXX_C;
+                break;
+            case LINE_XOOX_UNICODE:
+                sym = LINE_XOOX_C;
+                break;
+            case LINE_XXXO_UNICODE:
+                sym = LINE_XXXO_C;
+                break;
+            case LINE_XXOX_UNICODE:
+                sym = LINE_XXOX_C;
+                break;
+            case LINE_XOXX_UNICODE:
+                sym = LINE_XOXX_C;
+                break;
+            case LINE_OXXX_UNICODE:
+                sym = LINE_OXXX_C;
+                break;
+            case LINE_XXXX_UNICODE:
+                sym = LINE_XXXX_C;
+                break;
             default:
                 // sym goes unchanged
                 break;
         }
-        if( sym != 0 && sym < 256 ) {
+
+        if( static_cast<unsigned char>( sym ) != 0 && static_cast<unsigned char>( sym ) < 256 ) {
```

Comment:
```suggestion
        if( sym != 0 && sym < 256 ) {
```
These casts are not a good idea.  They might lead to inappropriate randomish misuse of fallback tiles.